src/main/resources/application.yaml

```diff
@@ -8,31 +8,7 @@ management:
   endpoints:
     web:
       base-path: /
-      exposure:
-        include: health, info, prometheus
```

Comment:
i think those are needed for pipeline interactions (in future). except `prometheus`